XRTK.Lumin/Packages/com.xrtk.lumin/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 XRTK
+Copyright (c) 2020 XRTK
```

Comment:
Also just a note that the actual package license and readme will always be overwritten with whatever is in the top level of the repo during the azure pipeline build process